documentation/modules/configuring-mtv-operator.adoc

```diff
@@ -66,6 +69,15 @@ spec:
 |`controller_block_overhead`
 |Fixed amount of additional space allocated in persistent block volumes. This setting is applicable for any `storageclass` that is block-based. It can be used when data, such as encryption headers, is written to the persistent volumes in addition to the content of the virtual disk. Note, this setting can only be changed using the CLI.
 |`0`
+
+
+|`vsphere_osmap_configmap_name`
+|Configuration map for vSphere source providers. An authorized user can, modify, or delete va;ues from the draft settings. Note, this setting can only be changed using the CLI.
+|Default settings for vSphere source providers
+
+|`ovirt_osmap_configmap_name`
+|Configuration map for {rhv-short} source providers. An authorized user can, modify, or delete va;ues from the draft settings. Note, this setting can only be changed using the CLI.
```

Comment:
i would add what the purpose of the map.
the map will map the incoming operation system of the VM to a fit preference on destination cluster.
the configmap should be in the namespace where forklift deployed (where the forklift-controller is running)

Reply:
@anarnold97 This is where "operation system" came from -- i have changed it to "operating system". Let me know if I need to make further changes to the text.